components/rp2040.rst

```diff
@@ -10,6 +10,9 @@ This component contains platform-specific options for the RP2040 platform.
 .. note::
 
     Support for all aspects of ESPHome on the RP2040 is still in development.
+    
+    Only the original model of Raspberry Pi Pico W board is supported, which has the Cypress **CYW43455** chip providing wireless connectivity. It can be identified by a metalic shield encapsulating the radio circuitry. Pico W boards having as radio modules chips like ESP8285 or similar (labelled as ``RP2040 Pico W-2023`` etc.), are not supported.
```

Comment:
```suggestion
    Only the original model of Raspberry Pi Pico W board is supported, which has the Cypress **CYW43455** chip providing wireless connectivity. It can be identified by a metallic shield encapsulating the radio circuitry. Pico W boards with radio module chips like ESP8285 or similar (labelled as ``RP2040 Pico W-2023`` etc.), are not supported.
```